app/views/components/profile-card.pug

```diff
@@ -9,7 +9,7 @@
   .profile__user-info
     span.profile__handle
       - if (user.github !== undefined)
-        a(href=user.github.html_url, target='_blank') @#{user.github.login}
+        |!{user.github.login}
```

Comment:
What does the `!` do here?

Reply:
You shouldn't need to escape HTML characters. Can you just try adding a space between the `|` and `#{user.github.login}`? So it should look like:

```
| #{user.github.login}
```

Reply:
Ok. I will do that and update the PR

Reply:
@robertcoopercode Done change it to `| #{user.github.login}`

Reply:
Thanks!